lib/index.js

```diff
@@ -1,6 +1,11 @@
-import { Selector } from 'testcafe';
+'use strict';
 
-export default Selector(complexSelector => {
+exports.__esModule = true;
+exports['default'] = void 0;
+
+var Selector = require('testcafe').Selector;
```

Comment:
`const { Selector } = require('testcafe')`

Ensure the [no-vars](https://eslint.org/docs/rules/no-var) rule.